docs/incident_attributes.rst

```diff
@@ -457,7 +457,7 @@ Microsoft SQL Server Login Attempt
 ===================================
 Triggered by any attempt to authenticate to the Microsoft SQL Server module.
 
-SQL Server supports multiple authentication modes, and the fields that come through depend on the mode.
+SQL Server supports multiple authentication modes and the fields that come through depending on the mode.
```

Comment:
We are talking about the fields that are returned in the event, and they "depend" (not "depending") on the mode.